tests/src/Kernel/EdtfUtilsTest.php

```diff
@@ -49,6 +49,7 @@ class EdtfUtilsTest extends KernelTestBase {
     '1900-01-02T01:22:33Z' => [],
     '1900-01-02T01:22:33+' => ['The date/time \'1900-01-02T01:22:33+\' is invalid.'],
     '1900-01-02T01:22:33+05:00' => [],
+    '1900/1999' => [],
```

Comment:
Is handled below: https://github.com/Islandora/controlled_access_terms/pull/118/files#diff-db5fc9b1d0aac630139ad5c4a17033653350c562c922c9b85624f8e97f5e0e2cR93

Reply:
Thanks - I [was just testing out something](https://github.com/Islandora/controlled_access_terms/actions/runs/9067225099/job/24911835713?pr=118) and this was easier than running the tests locally.

Reply:
... though, yeah looking bit more, guess it makes sense to test here for the validation, as it's more the extraction of an ISO 8601 value being tested below.